### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by ezmiller**
@thomastuts and @AnnJohn opening for review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,178 @@
...
+
+So what is a container component? The main thing to keep in mind here is that
+container components and presentation components go together. In fact, you can
+think of them as part of the same design pattern.  Where presentation components
+are dumb, container components are "smart". Where container components are
+usually subordinate "children" in a component hierarchy, container components
+are in almost every case the "parents" of presentational components.
+
+Here's a concise definition of the container component pattern:
+* Container components are primarily concerned with *how thing work*;
```
Comment (thomastuts) [inline README.md]: how **things** work

File: README.md [hunk 2]
```diff
@@ -1,23 +1,178 @@
...
+* they rarely have any html markup of their own, aside from a
```
Comment (thomastuts) [inline README.md]: 'html' -> 'HTML'

**Review by thomastuts — CHANGES_REQUESTED** [review]
Another great lesson! Two _very_ minor nits, but other than that, good to merge. Awesome job! 🚢

**Comment by AnnJohn** [comment]
@ezmiller could you update this and then merge? thanks!

**Pull request merged**